sympy/core/power.py

```diff
@@ -530,6 +530,8 @@ def _eval_is_extended_positive(self):
                 return log(self.base).is_imaginary
 
     def _eval_is_extended_negative(self):
+        if self.exp is S(1)/2:
+            return False
```

Comment:
In general the base can be any kind of expression so if we want to use reasoning based on complex numbers then we should check if the result is a complex number (e.g. `is_complex` or `is_extended_real` for `oo` and `-oo`)

These two should give None:
```
sqrt(zoo, evaluate=False).is_extended_negative 
sqrt(nan, evaluate=False).is_extended_negative 
```

An arbitrary symbol can be zoo or nan so we probably shouldn't have `negative` gives False in that case.

Reply:
```
>>> zoo.is_negative
False
```
I think  `sqrt(zoo).is_extended_negative` can be `False` as `zoo.is_negative` gives `False`.

Reply:
```
--- a/sympy/core/power.py
+++ b/sympy/core/power.py
@@ -531,7 +531,8 @@ def _eval_is_extended_positive(self):
 
     def _eval_is_extended_negative(self):
         if self.exp is S(1)/2:
-            return False
+            if self.base is not None:
+                return False
         if self.base.is_extended_negative:
             if self.exp.is_odd and self.base.is_finite:
                 return True

```
With this diff I get the following results:
```
>>> from sympy import *
>>> sqrt(nan).is_extended_negative
>>> sqrt(zoo).is_extended_negative
False
```